distro/rpm/openvpn.init.d.rhel

```diff
@@ -224,17 +224,24 @@ case "$1" in
 	fi
 	;;
   status)
-	if [ -f $lock ]; then
-	    for pidf in `/bin/ls $piddir/*.pid 2>/dev/null`; do
-		if [ -s $pidf ]; then
-		    kill -USR2 `cat $pidf` >/dev/null 2>&1
-		fi
-	    done
-	    echo "Status written to /var/log/messages"
-	else
-	    echo "openvpn: service not started"
-	    exit 1
-	fi
+        if [ -f $lock ]; then
+            for pidf in `/bin/ls $piddir/*.pid 2>/dev/null`; do
+                PID=$(cat $pidf)
+                if [ -e /proc/$PID ]; then
+                    echo "openvpn is running. pid: $PID"
+                    exit 0
+                else
+                    echo "openvpn is dead but lock file exists: $lock and pid file exists: $pidf"
+                    exit 1
+                fi
+            done 
```

Comment:
So this part loops through all pidfiles in $piddir, but exits with 0 or 1, depending on the status of the first OpenVPN process /bin/ls returns. So, if there are several OpenVPN instances running, the return value of the "service status" depends on the status of the first OpenVPN instance, which I think is suboptimal.

That said, I don't know if that is the best we can do, or what is the convention in these cases. Thoughts?